### ISSUE TITLE
Can not find `buf` with 0.6.0

### ISSUE THREAD

**Issue by tamasfe** [bufbuild/vscode-buf#334]
The extension seems to be always searching for `buf` inside the vscode workspace, most likely after #282.

```
Failed to get buf version: spawnSync /home/tamasfe/my-vscode-workspace/buf ENOENT
```

```sh
# which buf
/usr/bin/buf
```

**Comment by ahmednfwela** [comment]
also facing this on windows, where I use scoop to install buf

**Comment by paul-sachs** [comment]
Reverted relative path fix with 0.6.1, will investigate other ways to fix relative path issue.
